apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/setup-xm-cloud.md

```diff
@@ -0,0 +1,183 @@
+![XM Cloud Deploy Logs - Finished, Open XM Cloud Dashboard](https://sitecorecontenthub.stylelabs.cloud/api/public/content/51576832a2594fde8dd0fb7f86fbdfd1?v=59c586b8) 
+If you closed that screen already you can also access XM Cloud from the Sitecore Cloud Portal. Click the “Open app” button on the top right. 
+![Sitecore Cloud Portal - Open XM Cloud Dashboard](https://sitecorecontenthub.stylelabs.cloud/api/public/content/ef83f2ef7ff94688b9f84b59f57f98b5?v=8bed26f3) 
+2. Under the sites tab, click create site. 
+![Create new Site](https://sitecorecontenthub.stylelabs.cloud/api/public/content/08c2c56cb94c4ce8969c691a8b6ff4d3?v=c79d9844) 
+Here, you can choose to create an empty site, or a basic site that already has some content in it.  
+
+Note: when creating your own websites outside the scope of this blog or as a quick demo, we recommend using the empty site, as it requires less cleanup than the basic site. However, for this blog, you will select to use the basic site.   
+![Choose Site Template](https://sitecorecontenthub.stylelabs.cloud/api/public/content/1480ca22c0724a7496c9617b5f65cd6a?v=cf3753f9) 
+
+3. Name the site `Company DEV`.  
+![Name new Site and Site Collection](https://sitecorecontenthub.stylelabs.cloud/api/public/content/1dc9460c15fd452681187485fae6fe37?v=8feccf45) 
+
+4. Select a pre-configured language, in this case, English.  
+
+At this time, you can also select an existing Site identifier, or create a new one. This is required when connecting [CDP, the Customer Data Platform](https://developers.sitecore.com/customer-data-management/cdp) of Sitecore for embedded analytics and personalization.  
+
+Now, let’s start the creation process of the site.  
+
+5. Click `create website`, which will scaffold the website in the background. This will take a few minutes.  
+
+Once your site is scaffolded in the background, you can access it using XM Cloud Pages and start editing the content. 
+
+![Open Pages Editor](https://sitecorecontenthub.stylelabs.cloud/api/public/content/6438c1971da24b8f8564e3e48335d985?v=c5f57f17) 
+ 
+6. Click on the website, which opens up a WSYWIG editor where you can begin to edit the content, see components that you can use to edit the site, and switch between languages and your sites (top nav bar).   
+![In Pages Editor](https://sitecorecontenthub.stylelabs.cloud/api/public/content/9fb983d0eeec435797a1115f4ae801df?v=2cce57d4) 
+ 
+
+## Working locally  
+
+Navigate to your GitHub account and download the repository that was cloned earlier by the Sitecore XM Cloud Deploy App. 
+![GitHub - Clone Repository](https://sitecorecontenthub.stylelabs.cloud/api/public/content/239fa0b1ce314ca088bc830ee7a699e8?v=ba0d1b2e)
+
+### Download Repository
+
+1. In your repositories overview, find the newly created xm-cloud-tutorial-series repo.  
+2. Download the repository to your local file system using the GitHub CLI. Now, open the newly created folder in VS Code, or your text editor of choice.  
```

Comment:
I think we can remove "using the GitHub CLI" since we aren't specifying how to do that or depending on it. We can let the user download however they want to.

Reply:
Resolved in latest commit